WalletWasabi/BitcoinCore/P2pNode.cs

```diff
@@ -174,7 +174,7 @@ public void Disconnect()
 
 				try
 				{
-					Node?.Disconnect();
+					Node.Disconnect();
```

Comment:
In a concurrent scenario a property that is `not null` in a time `t` can become `null` in a time `t+1` so, there is no way to be sure. Even if you add `.?` in every single line it can fail anyway. For that reason, the pattern to use is:

* assign the property to a variable (ex: `var node = Node;`)
* if `node` is null you cannot do much with it but, if it is not null then it will never be null within that method